cmd/stack/deploy.go

```diff
@@ -47,6 +50,30 @@ func Deploy(ctx context.Context) *cobra.Command {
 				return err
 			}
 
+			if okteto.IsOkteto() {
+				c, err := okteto.NewOktetoClient()
+				if err != nil {
+					return err
+				}
+				hasAccess, err := utils.HasAccessToNamespace(ctx, s.Namespace, c)
+				if err != nil {
+					return err
+				}
+				if !hasAccess {
+					log.Warning("Namespace '%s' not found", s.Namespace)
+					create, err := utils.AskYesNo(fmt.Sprintf("Do you want to create the namespace '%s'? [y/n] ", s.Namespace))
```

Comment:
@rlamana should we change the color of this question to the purple one we are using on ctx command? This also affects when stignore/okteto manifest is not found

Reply:
Yes, all questions or input asked to the user should have the same format in the CLI.